docs/docker/building-net-docker-images.md

```diff
@@ -0,0 +1,189 @@
+microsoft/dotnet    1.0.0-core           b8da4a1fd280        253.2 MB
+```
+
+## Prerequisites
+
+- [.NET Core](http://dot.net)
+- Docker Engine : see [Docker Installation page](http://www.docker.com/products/docker)
+- Yeoman tooling for ASP.NET for creating the Web API application
+- Yeoman tools for Docker from Microsoft
+
+The Yeoman tools are installed using npm 
+
+```
+npm install -g yo generator-aspnet generator-docker
+```
+
+> [!NOTE]
+> This sample will be using [Visual Studio Code](http://code.visualstudio.com) for the editor.
+
+## Creating the Web API application
+
+Using the ASP.NET Yeoman generator, open a command or terminal session and type the following.
+```
+yo aspnet
+```
+
+Select **Web API Application** and type **api** for the name of the app ann tap enter.  Once the application is scaffolded, change to the `/api` directory and restore the NuGet dependencies using `dotnet restore`
+
+```
+cd api
+dotnet restore
+```
+
+Test the application using `dotnet run` and browsing to **http://localhost:5000/api/values**
+
+```javascript
+[
+    "value1",
+    "value2"
+]
+```
+
+Use `Ctrl+C` to stop the application.
+
+## Adding Docker support
+Adding Docker support the project is achieved using the Yeoman generator from Microsoft. It currently supports .NET Core, node.js and Golang projects by creating a Dockerfile and scripts that help build and run project inside containers. Visual Studio Code specific files are also added (launch.json, tasks.json) for editor debugging and command pallete support.
+
+```console
+$ yo docker
+
+     _-----_     ╭──────────────────────────╮
+    |       |    │   Welcome to the Docker  │
+    |--(o)--|    │        generator!        │
+   `---------´   │     Let's add Docker     │
+    ( _´U`_ )    │  container magic to your │
+    /___A___\   /│           app!           │
+     |  ~  |     ╰──────────────────────────╯
+   __'.___.'__
+ ´   `  |° ´ Y `
+
+? What language is your project using? (Use arrow keys)
+❯ .NET Core
+  Golang
+  Node.js
+
+```
+
+- Select `.NET Core` as the project type
+- `rtm` for the version of .NET Core
+- `Y` the project uses a web server
+- `5000` is the port the Web API application is listening on (i.e. http://localhost:5000)
+- `api` for the image name
+- `api` for the service name
+- `api` for the compose project 
+- `Y` to overwrite the current Dockerfile
+
+When the generator is complete the following files are added to the project
+
+- .vscode/launch.json
+- Dockerfile.debug
+- Dockerfile
+- docker-compose.debug.yml
+- docker-compose.yml
+- dockerTask.ps1
+- dockerTask.sh
+- .vscode/tasks.json
+
+The generator creates a two Dockerfiles.
+
+**Dockerfile.debug** - this file is based on the **microsoft/dotnet:1.0.0-preview2-sdk** image which if you note from the list of image variants, includes the SDK, CLI and .NET Core. Aptly named ".debug", will be the image used for debugging and development (F5) thus producing a larger image roughly 540MB.
```

Comment:
this part makes the sentence hard to read: which if you note from the list of image variants. I think it can be removed without losing value.
missing subject here: Aptly named ".debug", will be -> Aptly named ".debug", it'll be
Also, shouldn't this be inverted: debugging and development (F5) -> development and debugging (F5)